lib/galaxy_test/api/test_workflows.py

```diff
@@ -2964,6 +2964,12 @@ def test_cannot_run_against_other_users_history(self):
         run_workflow_response = self._post(f"workflows/{workflow_id}/invocations", data=workflow_request)
         self._assert_status_code_is(run_workflow_response, 403)
 
+    def test_cannot_run_bootstrap_admin_workflow(self):
+        workflow = self.workflow_populator.load_workflow(name="test_bootstrap_admin_cannot_run")
+        workflow_request, _ = self._setup_workflow_run(workflow)
```

Comment:
Looks like in 21.05 there were only 2 values to unpack but that changed to 3 in 21.09
```suggestion
        workflow_request, *_ = self._setup_workflow_run(workflow)
```

Reply:
Or maybe `workflow_request, *_ = self._setup_workflow_run(workflow)`? (unless that makes it look cryptic)

Reply:
Thanks, I didn't know that trick! it may be a little cryptic for initiates like me but it would certainly have avoided this failing test :laughing:

Reply:
As an initiate myself, I'm never sure whether shortcuts like that makes the code simpler or "more clever" :laughing: